pkg/config/token-server/config.go

```diff
@@ -7,14 +7,29 @@ import (
 
 func DefaultDataplaneTokenServerConfig() *DataplaneTokenServerConfig {
 	return &DataplaneTokenServerConfig{
-		Port: 5679,
+		Port:            5679,
+		PublicInterface: "",
+		PublicPort:      0,
+		TlsCertFile:     "",
+		TlsKeyFile:      "",
+		ClientCertFiles: nil,
 	}
 }
 
 // Dataplane Token Server configuration
 type DataplaneTokenServerConfig struct {
 	// Port of the server
 	Port uint32 `yaml:"port" envconfig:"kuma_dataplane_token_server_port"`
+	// Public interface on which the SSL server will be exposed
```

Comment:
I think, it would be more clear to group fields under `Local` and `Public`

Reply:
Changed, it is more clear.